contracts/helpers.sol

```diff
@@ -0,0 +1,74 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.4;
+
+import "./interfaces.sol";
+
+contract Helper {
+    ListInterface public immutable instaList;
+    ImplementationM1Interface public immutable instaImplementationM1;
+    InstaConnectorV2Interface public immutable instaConnectorV2;
+
+    constructor(
+        address _instaList,
+        address _instaImplementationM1,
+        address _instaConnectorV2
+    ) {
+        instaList = ListInterface(_instaList);
+        instaImplementationM1 = ImplementationM1Interface(
+            _instaImplementationM1
+        );
+        instaConnectorV2 = InstaConnectorV2Interface(_instaConnectorV2);
+    }
+
+    // DSA => manager address => ConnectorsInfo(connectors counter, connectors array, connectors mapping)
+    mapping(address => mapping(address => ConnectorsInfo))
+        public dsaManagerConnectors;
+    struct ConnectorsInfo {
+        uint256 connectorCount;
+        string[] allAddedConnectors;
```

Comment:
We might not want this array, because updating it on every add/remove of connectors might be very expensive. For the same reason, I haven't used it's length to get connector count of manager.

Hence, we can just keep an array of unique connector names on frontend side, and use connectorsEnabled of this Struct to know which connectors are present.

Reply:
It is better to create a map for this.
```
    struct ConnectorsInfo {
        mapping (string => bool) connectorAccess;
        }
```

Reply:
Already done. I was saying the same, 
https://github.com/Rushanksavant/InstaDapp-Task2/blob/b5ccb2e91f118c524311ae991ca6422b6ce35ea5/contracts/helpers.sol#L29

Reply:
So can we remove this array?
https://github.com/Rushanksavant/InstaDapp-Task2/blob/b5ccb2e91f118c524311ae991ca6422b6ce35ea5/contracts/helpers.sol#L28

Reply:
Yes we can remove this.